### PR TITLE
[REVIEW] Reformatting and updating CLX README

### PR DESCRIPTION
**Pull request by BartleyR**
Reformatted the CLX readme to flow better and updated the installations instructions to be more clear about the different ways to run the Docker container.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,11 +74,45 @@ pytest
...
+The notebooks folder contains example use cases and workflow instantiations.
+
+## Getting Started
+
+CLX is targeted towards cybersecurity data scientists, senior security analysts, threat hunters, and forensic investigators. Data scientists can use CLX in traditional Python files and Jupyter notebooks. CLX also includes structure in the form of a workflow. A workflow is a series of data transformations performed on a [GPU dataframe](https://github.com/rapidsai/cudf) that contains raw cyber data, with the goal of surfacing meaningful cyber analytical output. Multiple I/O methods are available, including Kafka and on-disk file stores.
+
+Example flow workflow reading and writing to file:
+
+```python
+from clx.workflow import netflow_workflow
```
Comment (brhodes10) [inline README.md]: A small note to consider moving code snippet closer to the top of the README and look into using a different code snippet example if needed

Comment (BartleyR) [reply README.md]: Good idea. I'll come up with a different code snippet and move this to the top to retain consistency across RAPIDS.

File: README.md [hunk 2]
```diff
@@ -1,56 +1,85 @@
...
+1. Accelerate log parsing in a flexible, non-regex method. and
+1. Provide SIEM integration with GPU compute environments via RAPIDS and effectively extend the SIEM environment.
 
-
-## Getting Started
-
-CLX is targeted towards cybersecurity data scientists, senior security analysts, threat hunters, and forensic investigators. Data scientists can use CLX in traditional Python files and Jupyter notebooks. CLX also includes structure in the form of a workflow. A workflow is a series of data transformations performed on a [GPU dataframe](https://github.com/rapidsai/cudf) that contains raw cyber data, with the goal of surfacing meaningful cyber analytical output. Multiple I/O methods are available, including Kafka and on-disk file stores.
-
-Example flow workflow reading and writing to file:
+## Getting Started with Python and Notebooks
```
Comment (BartleyR) [inline README.md]: @brhodes10 Can you review this new getting started section?

Comment (brhodes10) [reply README.md]: I like how `cuML` and a few other teams have a "bite-sized" code snippet highlighting the library itself and link to more fleshed out notebooks/use-cases following afterwards. Do you think this section should be more along the lines of "Getting Started with CLX" and give a more condensed code-snippet highlighting features in the CLX library?

Comment (BartleyR) [reply README.md]: This is a good idea. Going to move it into a new issue ([issue 112](https://github.com/rapidsai/clx/issues/112)).

File: README.md (hunk 1 above)
Comment (BartleyR) [reply README.md]: Moving this out to different issue for 0.13

**Comment by BartleyR** [comment]
@brhodes10 I pulled out your suggestions into [issue 112](https://github.com/rapidsai/clx/issues/112)

**Comment by BartleyR** [comment]
rerun tests

**Comment by BartleyR** [comment]
rerun tests

**Pull request merged**